kolibri/core/assets/src/mixins/commonCoreStrings.js

```diff
@@ -574,6 +583,10 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
     message: 'Explore',
     context: 'Resource and filter label for the type of learning activity. Translate as a VERB',
   },
+  activityType: {
+    message: 'Activity type',
+    context: 'Indicate the kind of type of learning activity',
```

Comment:
_kind of type_ sounds redundant... Maybe leave just the type and include examples: viewing, reading, etc.